LayoutTestBase/Core/Testers/LYTLayoutPropertyTester.m

```diff
@@ -113,7 +113,7 @@ + (UIView *)validateViewWithProviderProtocol:(Class)viewProvider
                                 limitResults:(LYTTesterLimitResults)limitResults
                                        block:(void(^)(UIView *view, NSDictionary *data, id context))validation {
     NSArray *sizes = [self viewSizesForProviderProtocol:viewProvider limitResults:limitResults];
-    if (sizes) {
+    if (sizes.count) {
```

Comment:
I'd prefer > 0 here rather than relying on the hidden cast. But I'll make that change myself since this pull request is very overdue.